### ISSUE TITLE
Error in import solution

### ISSUE THREAD

**Issue by lamimad** [seanmcne/Microsoft.Xrm.Data.PowerShell#271]
Using ver 2.3 on Dynamics 365 online I have some issues to import my CRM solution 
This is the command I'm running:
#Try importing with override customizations    
```
Import-CrmSolution -conn $connection -SolutionFilePath $solutionPath -MaxWaitTimeInSeconds 3600 -OverwriteUnManagedCustomizations -ActivatePlugIns -PublishChanges -Verbose
Write-Host ("Importing completed for CRM solution: {0} with file: {1}." -f $CRMSolutionName, $solutionPath)		
```	
The error message:

> 2018-01-04T16:30:53.4314314Z ##[error]Import-CrmSolution : Microsoft.PowerShell.Commands.WriteErrorException: ImportJob with ID: 
> d9723d88-d0d4-4760-8381-6594997391db has encountered an exception: The server was unable to process the request due to 
> an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from 
> ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the 
> exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and 
> inspect the server trace logs. 
> At ImportSolution.ps1:37 char:1
>  Import-CrmSolution -conn $connection -SolutionFilePath $solutionPath -MaxWaitTim ...
>  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>      CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
>      FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Import-CrmSolution
>  
> 
> 2018-01-04T16:30:54.4627165Z ##[error]Process completed with exit code 1 and had 1 error(s) written to the error stream

The vesrion of Microsoft.CrmSdk.CoreTools used is 8.2.0.5
I have this problem once on two, sometimes it works properly on the same solution.

**Comment by seanmcne** [comment]
Here's a couple of suggestions 
- Please update to the latest 2.6 or use the latest Main branch commit (as that will soon be released as v2.7).  
- Set your connection timeout using the included function/cmdlet: Set-CrmConnectionTimeout -TimeoutInSeconds 600
- use the async import parameter which will isolate you from network interruptions and timeouts

**Comment by seanmcne** [comment]
Please see #421
